soda-syncer/pkg/snapshot/worker.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021 The SodaFoundation Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http:#www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snapshot
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"net/http"
+	"os/exec"
+)
+
+type SnapshotProfile struct {
+	AwsAccesskey string `json:"AWS_ACCESS_KEY_ID"`
```

Comment:
This can work for any cloud and not specific to AWS, right? if we provide the AK, SK and bucket URL (as restic repository) then it can work for any cloud.

Reply:
If yes, then we can name the key as SECRET_KEY and ACCESS_KEY

Reply:
For each cloud the key is different, so we cannot generalize the key

Reply:
SODA multicloud bucket management works for most of the public cloud with AK/SK/URI. We can keep the key name generic and the values can be based upon

Reply:
For reference : https://restic.readthedocs.io/en/latest/040_backup.html#environment-variables

Reply:
got it thanks